commit 49703d8fcc47a0e7adcf97861525729d9e6ef539
Author: CSILLAG Tamas <cstamas@cstamas.hu>
Date:   Thu Dec 26 18:06:14 2013 +0100

    dehs: returned the repacked archive instead of the orig after file-excludes

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 6cb9c19..8bf6b99 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -1618,6 +1618,7 @@ EOF
 		$msg .= " and renamed it as $renamed_base";
 	    } elsif ($symlink eq 'files-excluded') {
 		$msg .= " and removed files from it in ${pkg}_${newversion}${excludesuffix}.orig.tar.$suffix\n";
+		$dehs_tags{'target'} = "${pkg}_${newversion}${excludesuffix}.orig.tar.$suffix";
 	    } else {
 		$dehs_tags{'target'} = $newfile_base;
 	    }
